portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,46 @@
 /**
```

Comment:
JS Doc is required for files as well. It should describe the contents of the file. Optionally it can also contain author's handle.

Example:

```
/**
  * This is the main javascript file for the page. Contains a blah blah method.
  * @author: sonudoo
  */
```